### ISSUE TITLE
GPU issues

### ISSUE THREAD

**Issue by tuvovan** [claroche-r/PnP_LADMM#4]
Hello,

I have a question regarding the GPU memory if the number of iteration is too big.
As set in your code the n_iters = 100, which mean that you use the denoiser 100 times, will it increase the memory of the GPU as well?

If not, how did you solve it?
thank you!

**Comment by claroche-r** [comment]
Hello,

I don't know if you figured it out but since we do not store the gradients(using with torch.no_grad()), the memory won't explode.

**Comment by tuvovan** [comment]
yeah I figured that out. I was having another question on your newest work but just realized that the number of blocks is now 8 instead of 100 :D 
thanks anyways
